### PR TITLE
Bug fix/internal issue 2035

### PR DESCRIPTION
**Pull request by hkernbach**
fixes: https://github.com/arangodb/planning/issues/2035#issuecomment-426994864

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js [hunk 1]
```diff
@@ -47,6 +71,24 @@
       });
     },
 
+    rerender: function () {
+      var self = this;
+
+      $.ajax({
+        type: 'GET',
+        cache: false,
+        url: arangoHelper.databaseUrl('/_api/engine'),
```
Comment (jsteemann) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js]: What is the purpose of calling `/_api/engine` repeatedly?

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js [hunk 2]
```diff
@@ -9,15 +9,39 @@
...
+        if (window.location.hash.indexOf('cIndices/' + self.collectionName) !== -1 && window.VISIBLE) {
+          if ($('#collectionEditIndexTable').is(':visible') && !$('#indexDeleteModal').is(':visible')) {
+            self.rerender();
+          }
+        }
+      }, self.refreshRate);
     },
 
+    interval: null,
+    refreshRate: 3000,
```
Comment (jsteemann) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js]: Do we really want to send requests to the database every 3 seconds?
I think we should avoid that. If we definitely want to repeatedly check for new indexes, we should change the frequency to something like at least 10 seconds.

Comment (hkernbach) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js]: Fine for me.

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js (hunk 1 above)
Comment (hkernbach) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js]: Thanks. Will fix.

**Pull request merged**